daffodil-core/src/test/scala/org/apache/daffodil/runtime2/TestRuntime2.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.daffodil.runtime2
+
+import scala.xml.Node
+import org.apache.daffodil.compiler.Compiler
+import org.junit.Test
+import java.io.ByteArrayInputStream
+
+import org.apache.daffodil.util.{SchemaUtils, Misc}
+import org.apache.daffodil.runtime2.generators.CodeGeneratorState
+
+class TestRuntime2 {
+
+  @Test def test1(): Unit = {
+
```

Comment:
This is our super-basic example. Just one root r, with one e1 element inside of type int.